engine/sound/src/sound.cpp

```diff
@@ -1041,13 +1041,17 @@ namespace dmSound
 
         dmSoundCodec::Info info;
         dmSoundCodec::GetInfo(sound->m_CodecContext, instance->m_Decoder, &info);
-        if (info.m_BitsPerSample == 16 && info.m_Channels > SOUND_MAX_MIX_CHANNELS ) {
-            dmLogError("Only mono/stereo with 16 bits per sample is supported (%s)", GetSoundName(sound, instance));
+        bool correct_bit_depth = info.m_BitsPerSample == 16 || info.m_BitsPerSample == 8;
```

Comment:
Not entirely sure how the previous check was thought to work. But I think the new version is clearer.

The available mixers are shown at line 942.